main_2024.py

```diff
@@ -159,7 +176,38 @@ def main() -> int:
             DATA_MERGE_TIMEOUT,
             detect_target_to_data_merge_queue,
             flight_interface_to_data_merge_queue,
-            data_merge_to_main_queue,
+            data_merge_to_geolocation_queue,
+            controller,
+        ),
+    )
+
+    camera_intrinsics = camera_properties.CameraIntrinsics.create(
+        GEOLOCATION_RESOLUTION_X,
+        GEOLOCATION_RESOLUTION_Y,
+        GEOLOCATION_FOV_X,
+        GEOLOCATION_FOV_Y,
+    )
+    camera_extrinsics = camera_properties.CameraDroneExtrinsics.create(
+        (
+            GEOLOCATION_CAMERA_POSITION_X,
+            GEOLOCATION_CAMERA_POSITION_Y,
+            GEOLOCATION_CAMERA_POSITION_Z,
+        ),
+        (
+            GEOLOCATION_CAMERA_ORIENTATION_YAW,
+            GEOLOCATION_CAMERA_ORIENTATION_PITCH,
+            GEOLOCATION_CAMERA_ORIENTATION_ROLL,
+        ),
+    )
```

Comment:
These create methods return a tuple - `True, object` if successful and `False, None` if unsuccessful.
Add a check for the result flag and return an error if the result is False